graph.cpp

```diff
@@ -116,6 +117,14 @@ vector<set<Node>> Graph::connected_components() {
 #ifdef VERIFY_SAMPLES_F
   verifier.verify_soln(retval);
 #endif
+
+  // the GraphWorkers may now process more updates
+  // However! because sketches are modified in place continuing
+  // without being careful will cause bad sketch behavior!
+  // So, only give more updates after CC if you have restored
+  // the sketch state from before CC
+  GraphWorker::unpause_workers();
```

Comment:
Another idea would be to have this unpause in its own graph function or to have some parameter to CC control whether to unpause. For now this seems like a reasonable approach.

Reply:
i think this is ok for now. if we're going to pause and restart stream ingestion, we'll probably want to call this as part of a graph function like we do with everything else